app/controllers/concerns/provider_website_redirect.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ProviderWebsiteRedirect
+  extend ActiveSupport::Concern
+
+  def provider_website
+    redirect_to provider.decorate.website, allow_other_host: true
```

Comment:
I assume you need to have this controller that will actually redirect to where the user wants because we only capture requests to our own controllers with dfe_analytics? We don't capture requests to outside services?

Reply:
Yeah, that's exactly right. We do this currently with the apply button on the course pages as well, I think it is a nice little solution without having to set up beacons or fire off other events (it looks like @tomas-stefano set it up for the apply button). The analysts will be able to sort for clicks to `course/:provider_code/:course_code:/provider_url` for instance, and know how many people visited a provider's website.

Reply:
Interesting, thank you 👍🏻

Reply:
Awesome idea. 🚀